moabb/datasets/braininvaders.py

```diff
@@ -704,3 +770,88 @@ def data_path(
         self, subject, path=None, force_update=False, update_path=None, verbose=None
     ):
         return _bi_data_path(self, subject, path, force_update, update_path, verbose)
+
+
+class VirtualReality(BaseDataset):
+    """
+    We describe the experimental procedures for a dataset that we have made publicly
+    available at https://doi.org/10.5281/zenodo.2605204 in mat (Mathworks, Natick, USA)
+    and csv formats. This dataset contains electroencephalographic recordings on 21
+    subjects doing a visual P300 experiment on PC (personal computer) and VR (virtual
+    reality). The visual P300 is an event-related potential elicited by a visual
+    stimulation, peaking 240-600 ms after stimulus onset. The experiment was designed
+    in order to compare the use of a P300-based brain-computer interface on a PC and
+    with a virtual reality headset, concerning the physiological, subjective and
+    performance aspects. The brain-computer interface is based on electroencephalography
+    (EEG). EEG data were recorded thanks to 16 electrodes. The virtual reality headset
+    consisted of a passive head-mounted display, that is, a head-mounted display which
+    does not include any electronics at the exception of a smartphone. A full description
+    of the experiment is available at https://hal.archives-ouvertes.fr/hal-02078533.
+
```

Comment:
Could you add the parameters for the dataset:

```suggestion
    Parameters
    ----------

    VR: bool (default True)
        if True, return runs corresponding to P300 experiment on virtual reality.

    PC: bool (default False)
        if True, return runs corresponding to P300 experiment on personal computer.
```

Reply:
I'm particularly a fan of snake case variables with a slightly more descriptive name, like `virtual_reality` or `personal_computer`, but I think it's more of a personal preference in this case.